cmd/kluctl/commands/utils.go

```diff
@@ -60,6 +60,14 @@ func withKluctlProjectFromArgs(ctx context.Context, projectFlags args.ProjectFla
 		}
 		repoOverrides = append(repoOverrides, ro)
 	}
+    for _, x := range projectFlags.LocalGitGroupOverride {
+		ro, err := parseRepoGroupOverride(x)
+		if err != nil {
+			return err
+		}
+		ro.IsGroup = true
```

Comment:
Better to put this into `parseRepoGroupOverride`

Reply:
ok, you know I had opposite idea :D